projects/Mallard/src/helpers/settings/defaults.ts

```diff
@@ -53,6 +53,11 @@ const notificationTrackingEndpoints = {
         'https://mobile-events.code.dev-guardianapis.com/notification/received',
 }
 
+export const senderId = {
```

Comment:
Might be nice to have a simple test to ensure we get the id expected in the associated environment  (can be a separate PR).